### PR TITLE
Update certification test depedencies

### PR DESCRIPTION
**Pull request by berndverst**
# Description

Updates certification tests to use latest `dapr/dapr` and removes any stale indirect dependencies that do not reflect the reality from `dapr/dapr` and `dapr/components-contrib`

### CODE REVIEW

**Comment by ItalyPaleAle** [comment]
@berndverst  is this still needed? If so, can you pls fix the merge conflicts and we'll merge this

**Review by ItalyPaleAle — DISMISSED** [review]

**Comment by ItalyPaleAle** [comment]
@berndverst please fix mod tidy

File: tests/certification/go.mod [hunk 1]
```diff
@@ -17,7 +17,7 @@ require (
 	github.com/cloudwego/kitex v0.5.0
 	github.com/cloudwego/kitex-examples v0.1.1
 	github.com/dapr/components-contrib v1.11.0-rc.1
-	github.com/dapr/dapr v1.10.5-rc.1.0.20230430203526-7a0fdf9f016b
+	github.com/dapr/dapr v1.10.7-0.20230510004949-770d4e51604f
```
Comment (ItalyPaleAle) [inline tests/certification/go.mod:20]: Should we use 1.11?

**Review by ItalyPaleAle — CHANGES_REQUESTED** [review]

File: tests/certification/go.mod (hunk 1 above)
Comment (berndverst) [reply tests/certification/go.mod:20]: this PR won't matter anymore when we merge release to master anyway

**Pull request closed without merging**